docs/metrics.md

```diff
@@ -24,6 +24,12 @@ Indicates whether the Software Emulation is enabled in the configuration. Type:
 ### kubevirt_console_active_connections
 Amount of active Console connections, broken down by namespace and vmi name. Type: Gauge.
 
+### kubevirt_memory_rss_to_requested_diff_bytes
+The pod with the highest exceeded memory for each container based on the rss. Type: Gauge.
+
+### kubevirt_memory_working_set_to_requested_diff_bytes
+The pod with the highest exceeded memory for each container based on the working set. Type: Gauge.
```

Comment:
The metric reports the diff between the pod with highest memory working set and its requested memory for each container, virt-controller, virt-handler, virt-api and virt-operator.

Reply:
fixed